source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -212,7 +212,7 @@ namespace PlayFab
                         if (!GetBinaryPayload(reqContainer, payload, payloadSize))
```

Comment:
>GetBinaryPayload [](start = 29, length = 16)

unrelated to your current change, but while reasoning about the flow of this code I inspected GetBinaryPayload and noticed that it was a no-op that always returns false making this code more complicated than it needs to be. If there's expected work to implement getbinarypayload I recommend adding a TODO to GetBinaryPayload. If not, I recommend making a work item to prune this unnecessary code